src/lib/operators/table_scan.cpp

```diff
@@ -32,7 +32,7 @@
 #include "table_scan/expression_evaluator_table_scan_impl.hpp"
 #include "table_scan/is_null_table_scan_impl.hpp"
 #include "table_scan/like_table_scan_impl.hpp"
-#include "table_scan/single_column_table_scan_impl.hpp"
+#include "table_scan/literal_table_scan_impl.hpp"
```

Comment:
You really liked `LiteralTableScanImpl` better than `Column(VS)ValueTableScanImpl`?

Reply:
Mmh? I am still planning to incorporate that change. Just wondered if it should be segment or column, see above.

Reply:
Mmh, since reference segments might result in multiple segments being scanned, I think I'll leave it as "column"